CSS/Customer/elementHolderMakeReservation.css

```diff
@@ -0,0 +1,63 @@
+.elementHolder {
+    position: fixed;
+    top:28%;
+    left:35%;
+    right:2%;
+    bottom:2%;
+    border-radius:40px;
+    background-color: floralwhite;
+    opacity: 0.6;
+}
```

Comment:
no need to repeat the styles in multiple files, if you have it in one, you can include it in any html file to be able to access it anywhere